assets/js/admin-script.js

```diff
@@ -105,6 +105,17 @@ jQuery( function( $ ) {
 		'delay':     200
 	} );
 
+	$( '#wpo-wcpdf-preview-wrapper #due_date' ).change( function() {
+		const $due_date_checkbox   = $( '#wpo-wcpdf-preview-wrapper #due_date' );
+		const $due_date_days_input = $( '#wpo-wcpdf-preview-wrapper #due_date_days' );
+
+		if ( $due_date_checkbox.is( ':checked' ) ) {
+			$due_date_days_input.prop( 'disabled', false );
+		} else {
+			$due_date_days_input.prop( 'disabled', true );
+		}
+	} ).trigger( 'change' );
+
```

Comment:
I think we also should provide some visual feedback, showing the field is disabled. Also the text in the field should align right so it looks better in combination with the word 'days' next to it.

<img width="293" alt="Scherm­afbeelding 2024-09-13 om 16 00 58" src="https://github.com/user-attachments/assets/a7c32480-8a53-44f1-9c7f-3dadcee5568e">

You should be able to achieve that with the following styles:

```
#wpo-wcpdf-preview-wrapper input#due_date_days {
	text-align: right;
}

#wpo-wcpdf-preview-wrapper input#due_date_days:disabled {
	background-color: #eaeaea;
	color: #999;
}
```